clusters/hive/pools/hypershift/hypershift-ocp-4-9-0-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -24,5 +24,5 @@ spec:
       region: us-east-1
   pullSecretRef:
     name: pull-secret
-  size: 4
+  size: 0
```

Comment:
I left `MaxSize` because we currently still have running tests, I will delete this tomorrow